Src/ILGPU/Runtime/OpenCL/CLAccelerator.cs

```diff
@@ -526,10 +527,19 @@ protected override CLKernel CreateKernel(CLCompiledKernel compiledKernel)
             MethodInfo launcher) =>
             new CLKernel(this, compiledKernel, launcher);
 
-        /// <summary cref="Accelerator.CreateStream"/>
+        /// <summary cref="Accelerator.CreateStream()"/>
         protected override AcceleratorStream CreateStreamInternal() =>
             new CLStream(this);
 
+        /// <summary cref="Accelerator.CreateStream(StreamFlags)"/>
+        protected override AcceleratorStream CreateStreamInternal(StreamFlags flag) =>
```

Comment:
ISSUE: Yeah, the more I think about it, let's not add support for `StreamFlags` on all accelerators - just CudaAccelerator. Then we dont have to import the `ILGPU.Runtime.Cuda` namespace.